grafana-plugin/src/components/MSTeams/MSTeamsInstructions.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { FC } from 'react';
```

Comment:
It seems that this component depends on our store, is connected to data, performs some backend calls etc. I think `components` folder is not the place where we want to keep such components, and we should rather use `containers` folder?